docs/theorymanual/decay.rst

```diff
@@ -74,6 +74,84 @@ in front of the summation:
 
     N_C(t) = N_1(0)  \gamma \left[1.0 - \sum_{i=1}^{C-1} \left(\prod_{j=1,i\ne j}^{C-1} \frac{\lambda_j}{\lambda_j - \lambda_i} \right) e^{-\lambda_i t} \right]
 
+Now, certain chains have intermeadiate nuclides that are *almost* stable. For example, decaying
+from Es-254 to Po-210 goes through U-238, which is very close to stable relative to all of the
+other nuclides in the chain. This can trigger floating point precision issues, where certain
+terms will underflow or overflow or generate NaNs. Obviously this is a situation to be avoided,
+if at all possible. To handle this sitiuation, let's call :math:`p` the index of the nuclide
+that is almost stable. We can then take the limit of the Bateman equations where
+:math:`\lambda_p \to 0` after we separate out the p-term from the summation:
+
+.. math::
+
+   \frac{N_C(t)}{N_1(0)} = \frac{\gamma}{\lambda_C}\sum_{i\ne p}^C \left[\lambda_i \frac{\lambda_p}{\lambda_p - \lambda_i}
+                                                        \left(\prod_{j\ne i,p}^C \frac{\lambda_j}{\lambda_j - \lambda_i}\right)
+                                                        e^{-\lambda_i t}\right]
+                           + \frac{\gamma}{\lambda_C} \lambda_p \left(\prod_{j\ne p}^C \frac{\lambda_j}{\lambda_j - \lambda_p} \right) e^{-\lambda_p t}
+
+   \lim_{\lambda_p\to 0}\frac{N_C(t)}{N_1(0)} = \frac{\gamma}{\lambda_C}\sum_{i\ne p}^C \lim_{\lambda_p\to 0} \left[\lambda_i \frac{\lambda_p}{\lambda_p - \lambda_i}
+                                                        \left(\prod_{j\ne i,p}^C \frac{\lambda_j}{\lambda_j - \lambda_i}\right)
+                                                        e^{-\lambda_i t}\right]
+                           + \lim_{\lambda_p\to 0} \frac{\gamma}{\lambda_C} \lambda_p \left(\prod_{j\ne p}^C \frac{\lambda_j}{\lambda_j - \lambda_p}\right) e^{-\lambda_p t}
+
+   \lim_{\lambda_p\to 0}\frac{N_C(t)}{N_1(0)} = \frac{\gamma}{\lambda_C}\sum_{i\ne p}^C \left[\lambda_i \frac{\lambda_p}{- \lambda_i}
+                                                        \left(\prod_{j\ne i,p}^C \frac{\lambda_j}{\lambda_j - \lambda_i}\right)
+                                                        e^{-\lambda_i t}\right]
+                           + \frac{\gamma}{\lambda_C} \lambda_p \left(\prod_{j\ne p}^C \frac{\lambda_j}{\lambda_j}\right) e^{-\lambda_p t}
```

Comment:
Is this right? You still have \lambda_p's left in the expression after removing the limit.

Reply:
Yes, I believe so, because `\lambda_p` isn't zero, it just almost zero and in some places that causes problems for floating point arithmetic.  So the limit is only taken for those reactions.  Said another way `\lambda_p << \lambda_j`

Reply:
So perhaps instead of using limits, one could write each step with \approx based on \lambda_p << \lambda_j.

Reply:
Yes, that would be better in some cases.  I'll rewrite soon.